src/app/pages/project/milestone/print-milestone/print-milestone.component.html

```diff
@@ -0,0 +1,84 @@
+<div [hidden]="isHidden" class="overlay mt-overlay modal fade" [ngClass]="{'show-modal': !this.isHidden}">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+
+      <div class="modal-header">
+        <h4 class="modal-title">{{title}}</h4>
+        <button type="button" class="close" aria-label="Close" (click)="hideModal()">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+
+      <div class="modal-body">
+        <div class="row">
+          <div *ngIf="pdf && !isHidden" class="col-sm-10 preview">
+            <iframe type="application/pdf" width='100%' height='500' [src]='pdf | safe'></iframe>
+          </div>
+          <div class="col-sm-2 line-margin">
+            <div class="col-sm-12 no-padding clearfix">
+              <mat-slide-toggle [(ngModel)]="showIssueList" (ngModelChange)="regenerate()">Issue List
+              </mat-slide-toggle>
+            </div>
+            <hr class="col-sm-12 no-padding">
+            <label class="pull-left col-sm-12 inline-label clearfix">Results:</label>
+            <div class="col-sm-12 no-padding clearfix">
```

Comment:
please, create

something like enum
class ReportOptionTypes
    status: string = "Status"
    resolution: string = "Resolution"

then create class ReportOption {
    type : ReportOptionTypes
    name : string
}

then create ReportOptions class
    failed :  ReportOption = new ReportOption(ReportOptionTypes.status, "failed")
    passed :  ReportOption = new ReportOption(ReportOptionTypes.status, "passed")
    appIssue :  ReportOption = new ReportOption(ReportOptionTypes.resolution, "app issue")

then class ReportToggle {
     option : ReportOptions
     state : boolean

then in the print-milestone.component.ts declare field like resultToggles : array[] = [new ReportToggle(ReportOptions.failed, true), new ReportToggle(ReportOptions.appIssue, false), ....]

in the print-milestone.component.ts

ReportToggle getToogle(ReportOptions option){
   // find necessary toggle within array of toggles
}


in the html use *ngFor=(toggle in toggles)...